typings/azdataGraph.d.ts

```diff
@@ -1,6 +0,0 @@
-declare module 'azdataGraph/dist/build' {
```

Comment:
Double-checking: this was intentionally removed, and doesn't need to be replaced?